p2p/host/resource-manager/limit_defaults.go

```diff
@@ -492,7 +492,7 @@ var DefaultLimits = ScalingLimitConfig{
 	},
 
 	PeerBaseLimit: BaseLimit{
-		ConnsInbound:    4,
+		ConnsInbound:    8,
```

Comment:
Can you add a comment explaining why this is 8 for now, and that smart dialing will allow us to reduce this back to 4?